srml/contract/src/account_db.rs

```diff
@@ -38,22 +41,28 @@ impl<T: Trait> Default for ChangeEntry<T> {
 	fn default() -> Self {
 		ChangeEntry {
 			balance: Default::default(),
-			code: Default::default(),
+			contract: Default::default(),
 			storage: Default::default(),
 		}
 	}
 }
 
+pub struct NewContract<T: Trait> {
+	code_hash: CodeHash<T>,
+	rent_allowance: BalanceOf<T>,
+}
```

Comment:
Why is it called `NewContract`? Isn't sometimes it would act as a data container for modified data of an existing contract?

Reply:
indeed I forget the update of rent_allowance mecanism.